src/lib/retire-js/RetireJsEngine.ts

```diff
@@ -185,16 +176,17 @@ export class RetireJsEngine extends AbstractRuleEngine {
 		return (await Promise.all(retireJsPromises)).reduce((all, next) => [...all, ...next], []);
 	}
 
-	private buildCliInvocations(rules: Rule[], target: string): RetireJsInvocation[] {
+	private buildRetireJsInvocations(rules: Rule[], target: string): RetireJsInvocation[] {
 		const invocationArray: RetireJsInvocation[] = [];
 		for (const rule of rules) {
 			switch (rule.name) {
 				case INSECURE_BUNDLED_DEPS:
 					// This rule is looking for files that contain insecure libraries, e.g. .min.js or similar.
-					// So we use --js and --jspath to make retire-js only examine JS files and skip node modules.
-					// We also hardcode a locally-stored vulnerability repo instead of allowing use of the default one.
+					// With version 4.x we use --jspath instead of --path to make retire-js only examine JS files and
+					// skip node modules. We also hardcode a locally-stored vulnerability repo instead of allowing use
+					// of the default one.
 					invocationArray.push({
-						args: [RetireJsEngine.RETIRE_JS_PATH, '--js', '--jspath', target, '--outputformat', 'json', '--jsrepo', RetireJsEngine.VULN_JSON_PATH],
```

Comment:
version 4 of retire doesn't have --js flag. Instead it makes --jspath and --path be separate where --jspath forces js only.